.github/workflows/machine-download.yaml

```
@@ -0,0 +1,69 @@
+name: machine-download
+on:
+    workflow_call:
```

pretend to use this as a workflow? if not, remove this line

```suggestion
    workflow_call:
```

---

Are we going to use changesets here?

This was this way because of changesets

---

I'm not sure. 

It won't hurt keeping it that, anyway.

---

In any case what is missing here is a release strategy, either manual tagging or changesets.

What is the strategy of helm? Manual or automated?

---

For the charts/, it's automated.

Tha tags are made by the CI after the charts hits the `main` branch, we have a release via GH Pages, to host the helm legacy way, and also use ghcr.io for the OCI Reistry way of releasing helm charts.

I think we should deprecate the old way, and keep only the OCI Registry option.

---

So does it make sense to let changesets take control of the “landing on main”, and delegate the release to a workflow call?

---

We'd have to test.

The `ct` tool used to release already tags, won't it conflict with the changests tagging process?